lua/entities/gmod_wire_expression2/core/custom/acffunctions.lua

```diff
@@ -52,9 +55,9 @@ local function GetLinkedWheels(Target)
 
 		Queued[Current] = nil
 		Checked[Current] = true
-
-		for Name, Action in pairs(Sources) do
-			for Entity in pairs(Action(Current)) do
+		
```

Comment:
Is this trailing whitespace or just git diffs tripping?

Reply:
This is trailing whitespace, it seems my linter broke.